bcrypt/xss.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+header('Content-Type: text/html; chatset=utf-8');
+
+echo $_GET['error'];
```

Comment:
Mishandled user input here leads to reflected XSS via raw dogging GET parameters into HTML (HTML injection). In this case, you want to use:
`html_entity_encode($input)`

You can include this function and use it like so:
```
require_once(APPROOT . '/encoding_util.php');
echo html_entity_encode($_GET['error']);
```